### PR TITLE
tweaks to UUID algorithm

### PR DESCRIPTION
**Pull request by bcoe**
What about this refactor that combines aspects I liked of the existing algorithm, with some of @marcoscaceres' suggestions?

Refs: #29

***
<a href="https://pr-preview.s3.amazonaws.com/WICG/uuid/pull/33.html" title="Last updated on Aug 21, 2021, 4:17 PM UTC (18f369e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/uuid/33/ca413d1...18f369e.html" title="Last updated on Aug 21, 2021, 4:17 PM UTC (18f369e)">Diff</a>

### CODE REVIEW

**Review by marcoscaceres — APPROVED** [review]
Super nice :D

**Comment by marcoscaceres** [comment]
I think the only think we might want to make super explicit in [=hexadecimal representation=] is that the "from" "to" numbers represent array index positions and are inclusive. It might be overkill tho - as it's pretty obvious how UUIDs are constructed, but just in case.

**Comment by domenic** [comment]
Not a fan, personally: https://github.com/WICG/uuid/issues/29#issuecomment-903983143

File: index.html [hunk 1]
```diff
@@ -113,41 +113,22 @@ <h2>
...
+        </li>
+        <li>
+          Let |timeHighAndVersion:string| be [=hexadecimal representation=] of |array| from 6 to 7.
+        </li>
+        <li>
+          Let |clockSeqAndReservedClockSeqLow:string| be [=hexadecimal representation=] of |array| from 8 to 9.
+        </li>
+        <li>
+          Let |node:string| be [=hexadecimal representation=] of |array| from 10 to 15.
+        </li>
```
Comment (marcoscaceres) [inline index.html:129]:
```suggestion
          Let |timeLow:string| be [=hexadecimal representation=] of |array| from 0 to 3, inclusive.
        </li>
        <li>
          Let |timeMid:string| be [=hexadecimal representation=] of |array| from 4 to 5, inclusive.
        </li>
        <li>
          Let |timeHighAndVersion:string| be [=hexadecimal representation=] of |array| from 6 to 7, inclusive.
        </li>
        <li>
          Let |clockSeqAndReservedClockSeqLow:string| be [=hexadecimal representation=] of |array| from 8 to 9, inclusive.
        </li>
        <li>
          Let |node:string| be [=hexadecimal representation=] of |array| from 10 to 15, inclusive.
        </li>
```

**Comment by bcoe** [comment]
Closing for time being so we can land #34.

**Pull request closed without merging**